### PR TITLE
Prevent password reset for migrated users  (fixes #1080)

### LINKED ISSUE #1080: Redirect /users/pwreset to https://accounts.firefox.com/reset_password
Redirect https://addons-dev.allizom.org/en-US/firefox/users/pwreset  to https://accounts.firefox.com/reset_password

### PR DESCRIPTION
**Pull request by mstriemer**
pwreset-migrated mov
> Migrated user

pwreset-unmigrated mov
> Unmigrated user

### CODE REVIEW

File: apps/users/templates/users/pwreset_request.html [hunk 1]
```diff
@@ -11,6 +11,16 @@
     <form method="post" class="listing-footer">
       {{ csrf() }}
       <fieldset>
+        {% if request.GET.migrated %}
```
Comment (kumar303) [inline apps/users/templates/users/pwreset_request.html]: I know that Django magic allows for `request.GET.migrated` but would `request.GET.get('migrated')` be ok?

Comment (kumar303) [inline apps/users/templates/users/pwreset_request.html]: Relying on `?migrated` seems a little awkward to me. What about re-checking `user.fxa_id` when this view loads and passing in `user_has_migrated` (or whatever) to the view template? A user who is monkeying with a query string probably can't do anything malicious but it seems like bad practice.

**Comment by kumar303** [comment]
r+wc

**Comment by mstriemer** [comment]
Definitely shouldn't have done the redirect. I was trying to avoid modifying the view but I could make the change in the form instead, so I did that.

**Comment by kumar303** [comment]
nice, that looks more straight forward

**Pull request merged**